app/src/main/java/com/quran/labs/androidquran/ui/helpers/HighlightType.java

```diff
@@ -8,20 +8,30 @@
 
 public class HighlightType implements Comparable<HighlightType> {
 
-  public static final HighlightType SELECTION = new HighlightType(1, false, R.color.selection_highlight);
-  public static final HighlightType AUDIO =     new HighlightType(2, false, R.color.audio_highlight);
-  public static final HighlightType NOTE =      new HighlightType(3, true,  R.color.note_highlight);
-  public static final HighlightType BOOKMARK =  new HighlightType(4, true,  R.color.bookmark_highlight);
+  public static final HighlightType SELECTION = new HighlightType(1, false, R.color.selection_highlight, false);
+  public static final HighlightType AUDIO =     new HighlightType(2, false, R.color.audio_highlight,     true);
+  public static final HighlightType NOTE =      new HighlightType(3, true,  R.color.note_highlight,      false);
+  public static final HighlightType BOOKMARK =  new HighlightType(4, true,  R.color.bookmark_highlight,  false);
 
   private Long id;
   private boolean multipleHighlightsAllowed;
   private int colorId;
   private Integer color = null;
+  private boolean floatable; 
+  // TODO: replace this with HighlightAnimationConfig
```

Comment:
considering your comment on 279 of HighlightingImageView and your comment 27 here, i'd go for making this class and only adding the things we need now - i.e. not future proofing it for everything, but making it easy to add say the type of interpolator there later if we want to change it, etc.

Reply:
Have implemented HighlightAnimationConfig which is flexible enough